tools/scripts/fuzz.sh

```diff
@@ -27,12 +27,17 @@ mkdir -p fuzz_corpus
 find test/testdata -iname "*.rb" | grep -v disable | xargs -n 1 -I % cp % fuzz_corpus
 mkdir -p fuzz_crashers/original
 
-echo "running"
 # use top 10 frames to tell different errors apart
 export ASAN_OPTIONS="dedup_token_length=10"
+# set FUZZ_ARGS to a space-delimited list of other args to pass to the fuzzer, for instance
+# --stress-incremental-resolver. by default it is '--'.
+FUZZ_ARGS="${FUZZ_ARGS:---}"
+
+echo "running with FUZZ_ARGS: $FUZZ_ARGS"
 nice "./bazel-bin/test/fuzz/$what" \
   -use_value_profile=1 \
```

Comment:
fyi: i'm no longer sure if this flag helps us. I think we should remove it.

Reply:
hmm, when I was pairing with @jvilk-stripe I believe he indicated that the `--stress-incremental-resolver` flag was still useful (and was in fact dismayed to learn that I had accidentally removed it in an earlier PR).

Reply:
I was talking about `-use_value_profile=1`, `--stress-incremental-resolver` is definitely useful for the other fuzzer